src/main/java/com/conveyal/taui/models/AnalysisRequest.java

```diff
@@ -119,17 +119,12 @@ public AnalysisTask populateTask (AnalysisTask task, Project project) {
         }
 
         // TODO define class with static factory function WebMercatorGridBounds.fromLatLonBounds(). Also include getIndex(x, y), getX(index), getY(index), totalTasks()
-
-        int east = Grid.lonToPixel(bounds.east, ZOOM);
-        int north = Grid.latToPixel(bounds.north, ZOOM);
-        int south = Grid.latToPixel(bounds.south, ZOOM);
-        int west = Grid.lonToPixel(bounds.west, ZOOM);
-
-        task.height = south - north;
-        task.north = north;
-        task.west = west;
-        task.width = east - west;
-        task.zoom = ZOOM;
+        Grid grid = new Grid(ZOOM, bounds.north, bounds.east, bounds.south, bounds.west);
```

Comment:
@abyrd and @ansoncfit we were technically a pixel off here. Most places account for the `+1` width/height seen in the R5 Grid constructor.

Reply:
Great to re-use existing code to convert from NSEW to width/height representation. It just eliminates and doubt over whether it's being done the same way everywhere.